text/0253-cdk-metadata-v2.md

```diff
@@ -0,0 +1,639 @@
+---
+feature name: CDK Metadata v2
```

Comment:
Nit: would be nice to name this RFC after the problem it's solving or the solution. Something like 'construct-usage-metadata'.